### ISSUE TITLE
How did you measure your FPS?

### ISSUE THREAD

**Issue by Jihyeon-2** [Dw1010/IntagHand#15]
Hello, 
First of all, thanks for sharing your nice job!

In the IntagHand paper, it says that your model has a performance of 30fps, so can you tell me where you measured it from?
Is it just time to get through the "network"?

I'm waiting for your response.
Thank you :)

**Comment by ligungun** [comment]
> Hello, First of all, thanks for sharing your nice job!
> 
> In the IntagHand paper, it says that your model has a performance of 30fps, so can you tell me where you measured it from? Is it just time to get through the "network"?
> 
> I'm waiting for your response. Thank you :)
Did you implement the training code? The accuracy of the code I implemented is quite different from the paper. Could you please make your training code public? thanks

**Comment by ligungun** [comment]
Did you implement the training code? The accuracy of the code I implemented is quite different from the paper. Could you please make your training code public? thanks
